botometerAnalyser/queues/retweeters.js

```diff
@@ -0,0 +1,41 @@
+const Bull = require('bull');
+const config = require('config');
+
+const { T } = require('../apis/twitter');
+
+
+const retweeterIdsQueueOptions = {
+	limiter: {
+		max: config.get('hooks.botometerAnalyser.twitter.rateLimits.STATUSES_RETWEETERS_IDS'),
+		duration: config.get('hooks.botometerAnalyser.twitter.rateLimits.TIME_WINDOW')
+	},
+	defaultJobOptions: {
+		removeOnComplete: true
+	}
+};
+if (process.env.NODE_ENV === 'test') {
+	retweeterIdsQueueOptions.redis = { db: 1 };
+}
+const retweeterIdsQueue = new Bull('Twitter: GET statuses/retweeters/ids', retweeterIdsQueueOptions);
+
+function getRetweetersIds(tweetId) {
+	const twitterParams = {
+		id: tweetId,
+		count: 100,
```

Comment:
We should use the `cursor` to get other followers and not only 100

Reply:
Unfortunately this is not supported by the twitter API (even though it is documented):

> The response from the API will include a previous_cursor and next_cursor to allow paging back and forth. See our cursor docs for more information.
> 
> While this method supports the cursor parameter, the entire result set can be returned in a single cursored collection. Using the count parameter with this method will not provide segmented cursors for use with this parameter.

See https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/get-statuses-retweeters-ids.html and https://github.com/sferik/twitter/issues/425

... yes, I could not believe it.

Reply:
Can you check if we can use [`GET statuses/retweets/:id`](https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/get-statuses-retweets-id.html) instead ? Maybe pagination will work with this endpoint